### ISSUE TITLE
Accessibility 2025

### ISSUE THREAD

**Issue by nrllh** [HTTPArchive/almanac.httparchive.org#4063]
# Accessibility 2025

![Accessibility illustration](https://raw.githubusercontent.com/HTTPArchive/almanac.httparchive.org/main/src/static/images/2019/accessibility/hero_lg.jpg)

If you're interested in contributing to the Accessibility chapter of the 2025 Web Almanac, please reply to this issue and indicate which role or roles best fit your interest and availability: [author](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide), [reviewer](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Reviewers'-Guide), [analyst](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Analysts'-Guide), and/or [editor](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Editors'-Guide).  You can find more details about this year’s Call for Contributions [here](https://github.com/HTTPArchive/almanac.httparchive.org/discussions/4062).

💬 Please make sure to join [#web-almanac-accessibility](https://join.slack.com/t/httparchive/shared_invite/zt-45sgwmnb-eDEatOhqssqNAKxxOSLAaA) on Slack for team coordination.

## Content team

| Lead | Authors | Reviewers | Analysts | Editors | Coordinator |
| ---- | ------- | --------- | -------- | ------- |    -------  |
|  -  | - | - | - | -  | - |

<details>
<summary><b>Expand for more information about each role 👀</b></summary>

- The **[content team lead](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Content-Team-Leads'-Guide)** is the chapter owner and responsible for setting the scope of the chapter and managing contributors' day-to-day progress.
- **[Authors](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide)** are subject matter experts and lead the content direction for each chapter. Chapters typically have one or two authors. Authors are responsible for planning the outline of the chapter, analyzing stats and trends, and writing the annual report.
- **[Reviewers](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Reviewers'-Guide)** are also subject matter experts and assist authors with technical reviews during the planning, analyzing, and writing phases.
- **[Analysts](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Analysts'-Guide)** are responsible for researching the stats and trends used throughout the Almanac. Analysts work closely with authors and reviewers during the planning phase to give direction on the types of stats that are possible from the dataset, and during the analyzing/writing phases to ensure that the stats are used correctly.
- **[Editors](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Editors'-Guide)** are technical writers who have a penchant for both technical and non-technical content correctness. Editors have a mastery of the English language and work closely with authors to help wordsmith content and ensure that everything fits together as a cohesive unit.
- The **[section coordinator](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Section-Leads'-Guide)** is the overall owner for all chapters within a section like "User Experience" or "Page Content" and helps to keep each chapter on schedule.

_Note: The time commitment for each role varies by the chapter's scope and complexity as well as the number of contributors._

For an overview of how the roles work together at each phase of the project, see the [Chapter Lifecycle](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle) doc.
</details>

### [0. Form the content team](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#0-create-content-team)
- [ ] 📆 `May 18` Complete program and content committee - 🔑 *Organizing committee*
  - The content team has at least one author, reviewer, and analyst.

### [1. Plan content](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#1-plan-content)
- [ ] 📆 `June 1` **First meeting to outline the chapter contents** - 🔑 *Content team*
  - The content team has completed the chapter outline.

### [2. Gather data](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#2-gather-data)
- [ ] 📆 `July 1` **Custom metrics completed** - 🔑 *Analysts*
  - Analysts have added all necessary [custom metrics](https://github.com/HTTPArchive/custom-metrics/blob/main/README.md) and drafted a PR ([example](https://github.com/HTTPArchive/almanac.httparchive.org/pull/1087)) to track query progress. 
- [ ]  📆 `July 1` **HTTP Archive Crawl** - 🔑 *HA Team*
  - HTTP Archive runs the June crawl.

### [3. Validate results](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#3-validate-results)
- [ ] 📆 `September 1` **Query Metrics & Save Results** - 🔑 *Analysts*
  - Analysts have queried all metrics and saved the output.

### [4. Draft content](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#4-draft-content)
- [ ] 📆 `October 1` **First Draft of Chapter** - 🔑 *Authors*
  - Authors has written the chapter. 
- [ ] 📆 `October 20` **Review & Edit Chapter** - 🔑 *Reviewers & Editors*
  - Reviewers and  Editors has processed the the chapter.

### [5. Publication](https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Chapter-Lifecycle#5-publication)
- [ ] 📆 `November 15` **Chapter Publication (Markdown & PR)** - 🔑 *Authors*
  - Authors has converted the chapter to markdown and drafted a PR.
- [ ] 📆 `December 1` **Launch of 2025 Web Almanac 🚀** - 🔑 *Organizing committee*
 
### [6. Live Stream](#)
- [ ] 📆 `December 15` **Live Stream** - 🔑 *Content Team*

<a id="chapter-resources"></a>
## [Chapter resources](#chapter-resources)
*Google Docs, Sheets, and other resources will be added soon.*

**Comment by mgifford** [comment]
I'm happy to contribute again this year. I'd love to have some more support though.

**Comment by jpsherman** [comment]
I'm more than happy to contribute - at Red Hat, search and accessibility exist in the same team. I'm comfortable as a reviewer or an analyst.

**Comment by ninaroby** [comment]
I'm also happy to contribute as either a reviewer or an analyst.

**Comment by SaptakS** [comment]
@mgifford I would love to co-author. I was one of the authors for this chapter in 2022.

**Comment by tricinel** [comment]
I'd love to get involved in this! I'm comfortable as an author or an editor. I've been writing a daily newsletter about web accessibility for the past 435 days - haven't missed a day yet. :)

**Comment by mikaelaraujo** [comment]
@tricinel Welcome!

**Comment by beckycolley** [comment]
Hi, I'd love to get involved with this as an editor if you'll have me!

**Comment by mikaelaraujo** [comment]
> Hi, I'd love to get involved with this as an editor if you'll have me!

For sure @beckycolley! Welcome.

**Comment by SaptakS** [comment]
I think @mgifford was also interested in co-authoring the chapter? I have lead the chapter before once, so I would prefer someone else leads. Otherwise I am happy to do it.

**Comment by nrllh** [comment]
@SaptakS, could you please try to schedule a meeting with the content team to draft this year's outline?

**Comment by tricinel** [comment]
If I can help in any way with organising things, I'd love to get involved.

**Comment by SaptakS** [comment]
@tricinel we would love to have your contribution. Is there a particular role you are more interested in?

**Comment by SaptakS** [comment]
@jpsherman @ninaroby @mgifford @tricinel @beckycolley if you are not already in the [slack channel](https://join.slack.com/t/httparchive/shared_invite/zt-45sgwmnb-eDEatOhqssqNAKxxOSLAaA), can you please join, so we can coordinate and schedule a meeting?

**Comment by tricinel** [comment]
@SaptakS I already signed up as an editor. I was just asking if you need additional support with organising things and whatnot :)

**Comment by SaptakS** [comment]
Ah I see. I mean I would still prefer someone else other than me lead the chapter (which is basically some organizing stuff like scheduling meetings, or ensuring that the chapter meets all the deadlines). Would you be interested to be the chapter lead maybe?

**Comment by tricinel** [comment]
Sorry for the late reply. June and July are pretty much a no-go (family). But I can help from mid-August on-wards.

**Comment by mikaelaraujo** [comment]
@SaptakS We need to schedule a call to define the chapter's deadline. After that, I can support you if you need.

**Comment by mehm8128** [comment]
Hi, I looked over the preview of accessibility section and found the display bug on "Map of accessible countries by Top Level Domain (TLD)".
When I hover on Japan, "edu avg_accessibility_score:" is shown and Japan is not painted by any color (maybe because I'm in Japan?).
The TLD corresponds to Japan is `jp`, so I think it should be "jp avg_accessibility_score: 81%" and painted by blue ([ref](https://docs.google.com/spreadsheets/d/13sY_wmYODArxo-hH5cSuRAbvtLdGI3x5Xc9qUyqP8as/edit?gid=1037208406#gid=1037208406&range=S125)).

<img width="781" height="547" alt="Map of accessible countries by Top Level Domain (TLD). Japan is not painted and the text 'edu avg_accessibility_score:' is shown nearby Japan" src="https://github.com/user-attachments/assets/59e004b3-4ac7-4ac2-b229-c9bea16e8aeb" />

`edu` is on the O292 cell on the Google SpreadSheet.
<img width="335" height="218" alt="edu on the O292 cell" src="https://github.com/user-attachments/assets/500e84a9-c289-4bfc-a4a3-883302a68a45" />

https://docs.google.com/spreadsheets/d/13sY_wmYODArxo-hH5cSuRAbvtLdGI3x5Xc9qUyqP8as/edit?gid=1037208406#gid=1037208406&range=O292

**Comment by tunetheweb** [comment]
Checked and updated. Can you confirm it's OK from your side at https://accessibility-2025-dot-webalmanac.uk.r.appspot.com/en/2025/accessibility?print#fig-35

**Comment by mehm8128** [comment]
@tunetheweb 
Thank you, I confirmed it is correctly displayed!
